### PR TITLE
stars

### PR DESCRIPTION
**Pull request by DenisGurskiy**
https://DenisGurskiy.github.io/layout_stars/
https://DenisGurskiy.github.io/layout_stars/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,47 @@
...
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul class="star stars--0">
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+    </ul>
```
Comment (VitaliyBondarenko1982) [inline src/index.html:17]: it shouldn't be a list. 
make all elements div
and you will not need to reset default list styles

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  background-image: url(images/star-active.svg);
+}
+
+.star:nth-of-type(n+5) > .stars__star:nth-of-type(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.star:last-of-type > .stars__star:last-of-type {
+  background-image: url(images/star-active.svg);
+} */
```
Comment (VitaliyBondarenko1982) [inline src/style.css:62]: remove commented code

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 > .stars__star:nth-of-type(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 > .stars__star:nth-of-type(-n+4) {
+  background-image: url(images/star-active.svg);
+}
```
Comment (VitaliyBondarenko1982) [inline src/style.css:42]:
```suggestion
.star > .stars__star {
  background-image: url(images/star.svg);
}

.stars--5 > .stars__star,
.stars--1 > .stars__star:nth-of-type(-n+1),
.stars--2 > .stars__star:nth-of-type(-n+2),
.stars--3 > .stars__star:nth-of-type(-n+3),
.stars--4 > .stars__star:nth-of-type(-n+4) {
  background-image: url(images/star-active.svg);
}
```
separate selectors with same styles by comma

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 4]
```diff
@@ -8,6 +8,47 @@
...
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="star stars--1">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
```
Comment (nazarmatsevych) [inline src/index.html:24]:
```suggestion
    <div class="star stars--0">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>

    <div class="star stars--1">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>
```

Add blank lines between multiple sibling blocks

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,29 @@
+* {
+  margin: 0;
+}
```
Comment (nazarmatsevych) [inline src/style.css:3]: Don't use * selector for zeroing margins or paddings

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Almost done, 
pls check comments

**Review by vlad-demchuk — APPROVED** [review]

**Pull request closed without merging**